frontend/src/__tests__/cypress/cypress/tests/mocked/hardwareProfiles/legacyHardwareProfiles.cy.ts

```diff
@@ -570,27 +570,6 @@ describe('legacy profiles table', () => {
         }
       });
     });
-
```

Comment:
There seems to be an issue with retrieving visibility and sources from the legacy hwp table but since the table will be removed I removed these tests.